<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecordsDirImport</class>
 <widget class="QWidget" name="RecordsDirImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批量导入</string>
  </property>
  <widget class="QProgressBar" name="pb_curStep">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>100</y>
     <width>541</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar::chunk {   background-color: #1296db;   width: 20px;}
QProgressBar { text-align: center;}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_runninginfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>74</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>test.xlsx</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_currentstep">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>74</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1/24</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_startBatchimport">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>140</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/excute.png</normalon>
    </iconset>
   </property>
  </widget>
  <widget class="QLabel" name="le_batchDir">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>391</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_chooseBatchDir">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="BidsManager.qrc">
     <normaloff>:/folder.png</normaloff>:/folder.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>文件目录：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BidsManager.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_chooseBatchDir</sender>
   <signal>clicked()</signal>
   <receiver>RecordsDirImport</receiver>
   <slot>onDirChooseSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_startBatchimport</sender>
   <signal>clicked()</signal>
   <receiver>RecordsDirImport</receiver>
   <slot>onStartBatchImportSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>533</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>390</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDirChooseSlot()</slot>
  <slot>onStartBatchImportSlot()</slot>
 </slots>
</ui>
